### DISCUSSION TITLE
[Feature Request]: show certificate expire information

### DISCUSSION THREAD

**Discussion by robindirksen1** [beyondcode/herd-community#361] [Ideas]
### Feature Description

Last year, I installed a project and secured it. Now, after migrating from Valet to Herd, the certificate expired (1 year ago).

It would be useful to see the status (`expired`, or `valid for xxx days`) of the certificate in Herd sites overview.
Also, maybe show an alert when the certificate is expired to recreate a (new) certificate.

![Screenshot 2023-11-27 at 12 02 26@2x](https://github.com/beyondcode/herd-community/assets/22446895/fc62d84b-2fa8-40f2-8a10-a53adec60451)

**Comment by mpociot** [comment]
Good idea. Might be nice to just auto-renew them as they are just local certificates

**Comment by robindirksen1** [comment]
@mpociot agree on auto-renew!

ps, still trying to "renew" the certificate (with subdomains) but it keeps untrusted in my Chrome browser.
